template/scala-akka-http-server/api.mustache

```diff
@@ -48,8 +49,9 @@ import akka.http.scaladsl.server.directives.FileInfo
 
    private def logHttp(ctxs: Seq[(String, String)]): Directive0 = {
        val contexts = ctxs.toMap
+       val subject   = contexts.get(UID).filter(!_.isBlank).orElse(contexts.get(SUB))
```

Comment:
https://github.com/pagopa/interop-be-agreement-process/pull/56#discussion_r813967484